tests/autorun-test.ts

```diff
@@ -5,6 +5,14 @@ import lolex from 'lolex';
 const SET_TIMEOUT = setTimeout;
 let fakeClock;
 
+function flushMicrotasks() {
```

Comment:
isn't this flush macro task, as the promise wont resolve until `setTimeout(resolve, 0)`

Reply:
It’s to ensure the current event frame’s microtask queue is flushed by resolving in a macro task

Reply:
Does that make sense @stefanpenner, I can rename if you'd prefer still just give me an alternate...

Reply:
ya, I understand what it does.

The name is off, it isn’t flush microtasks, it’s wait for some near future macro task.

Nothing prevents other unexpected work from sneaking in between so the “flush” word is misleading

Reply:
happy to change it, what would you like it to be named?

Some possibilities:

* `escapeMicrotaskQueue`
* `stefsMagicalFutureMacroTask` (😜)
* `waitForNextMacroTask`

Reply:
OK, I just renamed to `escapeCurrentMicrotaskQueue` which actually seems a bit more descriptive as well.